### ISSUE TITLE
Typing is not working properly

### ISSUE THREAD

**Issue by daveb-501commons** [microsoft/vscode#86966]
Issue Type: <b>Bug</b>

I just applied the update and now typing isn't working properly.  When I start typing the cursor goes back a character and then forward.  So for it to work you type a character, cursor moves to left of character then back to the right of the character - this takes about 2 seconds.  If you type faster than 2 seconds a character then the order of all the characters gets messed up because each new character is inserted before the previous character.

I've reboot machine and still getting the issue.

VS Code version: Code 1.41.0 (9579eda04fdb3a9bba2750f15193e5fafe16b959, 2019-12-11T18:37:42.077Z)
OS version: Windows_NT x64 10.0.17763

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz (8 x 2808)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off<br>surface_control: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.55GB (22.04GB free)|
|Process Argv|.|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (15)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-eslint|dba|1.9.1
bower|don|0.0.3
prettier-vscode|esb|3.11.0
mssql|ms-|1.7.1
python|ms-|2019.11.50794
vscode-xml|red|0.10.0
salesforcedx-vscode|sal|47.10.0
salesforcedx-vscode-apex|sal|47.10.0
salesforcedx-vscode-apex-debugger|sal|47.10.0
salesforcedx-vscode-apex-replay-debugger|sal|47.10.0
salesforcedx-vscode-core|sal|47.10.0
salesforcedx-vscode-lightning|sal|47.10.0
salesforcedx-vscode-lwc|sal|47.10.0
salesforcedx-vscode-visualforce|sal|47.10.0
sfdx-cli|vib|0.0.1

</details>

**Comment by IllusionMH** [comment]
/needsMoreInfo

Is it reproducible without extensions?
You can try this with <kbd>F1</kbd> and `>Developer: Reload Window With Extensions Disabled`

Are you typing in editor or in terminal?

**Comment by cvasani** [comment]
> /needsMoreInfo
> 
> Is it reproducible without extensions?
> You can try this with F1 and `>Developer: Reload Window With Extensions Disabled`
> 
> Are you typing in editor or in terminal?

Same issue for me also. This was not a case before upgrade.

**Comment by cvasani** [comment]
> vscode-xml

For me XML was causing this issue. Could you please give a try ?

**Comment by TomerFi** [comment]
This is the case for me too immediately after upgrading to  1.41.0.
It doesn't matter which direction the typing is advancing in...
Even if I'm not really typing and just going thru a line with the `arrow keys`:
If I go left to right, every couple of characters the marker goes back one character backward (the left side of the line).
And if I go right to left, every couple of characters the marker goes back one character backward (the right side of the line).

**Comment by daveb-501commons** [comment]
More Info Requested

I reloaded with all Extensions disabled and the problem was fixed

I narrowed it down to XML Red Hat Extension.  When I disabled this extension everything works.  When I enable this extension after about 5 seconds the issue surfaces again.  They have an extension upgrade as of 12/13/2019 which I suspect is causing the issue.

Entered the following issue on XML Red Hat GitHub repository
https://github.com/angelozerr/lsp4xml/issues/601
